apps/maestro/src/lib/hooks/useConnectWallet.ts

```diff
@@ -49,14 +49,14 @@ export function useConnectWallet() {
     return false;
   };
 
-  const connectWallet = async ({ chainId }: { chainId: number }) => {
+  const connectWallet = ({ chainId }: { chainId: number }) => {
     const isTargetChainSui = chainId === suiChainConfig.id;
 
     if (isTargetChainSui) {
       return tryConnectSuiWallet();
     } else {
       setPendingChainId(chainId);
-      await openWeb3Modal();
+      void openWeb3Modal();
```

Comment:
why do you need `void` keyword? i've never been using that. how does it different than removing it?